carbonmark-api/src/routes/activities/get.utils.ts

```diff
@@ -0,0 +1,40 @@
+import { Static } from "@sinclair/typebox";
+import { FastifyInstance, FastifyReply } from "fastify";
+import { mapValues } from "lodash";
+import { split } from "lodash/fp";
+import { ActivityType } from "../../.generated/types/marketplace.types";
+import { Activity } from "../../models/Activity.model";
+import { NetworkParam } from "../../models/NetworkParam.model";
+import { gql_sdk } from "../../utils/gqlSdk";
+import { fetchProjectActivities } from "../../utils/helpers/fetchMarketplaceListings";
+import { stringsToActivityTypes } from "../../utils/helpers/utils";
+import { schema } from "./get.schema";
+
+/**
+ * This handler fetches activities for a carbon project
+ *
+ * @returns
+ */
+export const getActivities = async (
```

Comment:
Thoughts on having this util function return the activities and let the routes handle replies/returning the data? There would be no need to pass the fastify instance and reply param to this function?

For example:
```
export const getActivities = async (
  query: Static<typeof schema.querystring>,
  network?: NetworkParam
): Promise<Activity[]> => {
  // ...
  return await fetchProjectActivities(sdk, { ... });
};
```

And then in the route files:

```
const handler = (fastify: FastifyInstance) =>
  function (
    request: FastifyRequest<{
      Querystring: Static<typeof schema.querystring>;
    }>,
    reply: FastifyReply
  ): Promise<Activity[]> {
    const activities = getActivities(request.query, request.query.network);
    return reply.status(200).header("Content-Type", "application/json; charset=utf-8").send(activities);
  };
  ```